examples/campaign_management/create_campaign_experiment.py

```diff
@@ -33,35 +36,30 @@ def main(client, customer_id, campaign_draft_resource_name):
         uuid.uuid4())
     campaign_experiment.traffic_split_percent.value = 50
     campaign_experiment.traffic_split_type = client.get_type(
-        'CampaignExperimentTrafficSplitTypeEnum',version='v1').RANDOM_QUERY
+        'CampaignExperimentTrafficSplitTypeEnum', version='v1').RANDOM_QUERY
 
     try:
         campaign_experiment_service = client.get_service(
-            'CampaignExperimentService',
-            version='v1'
-        )
+            'CampaignExperimentService', version='v1')
 
         # A Long Running Operation (LRO) is returned from this
         # asynchronous request by the API.
         campaign_experiment_lro = (
             campaign_experiment_service.create_campaign_experiment(
-                customer_id,
-                campaign_experiment
-            )
-        )
-    except google.ads.google_ads.errors.GoogleAdsException as ex:
-        print('Request with ID "%s" failed with status "%s" and includes the '
-              'following errors:' % (ex.request_id, ex.error.code().name))
+                customer_id, campaign_experiment))
+    except GoogleAdsException as ex:
+        print('Request with ID "{}" failed with status "{}" and includes the '
+              'following errors:'.format(ex.request_id, ex.error.code().name))
         for error in ex.failure.errors:
-            print('\tError with message "%s".' % error.message)
+            print('\tError with message "{}".'.format(error.message))
             if error.location:
                 for field_path_element in error.location.field_path_elements:
-                    print('\t\tOn field: %s' % field_path_element.field_name)
+                    print('\t\tOn field: {}'.format(field_path_element.field_name))
         sys.exit(1)
 
     print('Asynchronous request to create campaign experiment with '
-          'resource name "%s" started.'
-          % campaign_experiment_lro.metadata.campaign_experiment)
+          'resource name "{}" started.'.format(
+            campaign_experiment_lro.metadata.campaign_experiment))
```

Comment:
Add another tab indent here, this should be 4 spaces like on line 74.

Reply:
Done